darwin.c

```diff
@@ -7,82 +7,8 @@
 #include <mach-o/dyld.h>
 #include "common.h"
 
-int elevate(char *path, char *prompt, char *iconPath)
```

Comment:
[cocoasudo](https://github.com/performantdesign/cocoasudo) already takes care of this for us, so there's no need for us to maintain this code.